package.json

```diff
@@ -22,7 +22,11 @@
     "build-storybook": "build-storybook -c .storybook -o news-theme-blocks",
     "test:coverage": "jest --coverage",
     "generate:intl": "node locale/scripts/generate-intl.js",
-    "postinstall": "npx lerna clean -y"
+    "postinstall": "npx lerna clean -y",
+    "release:beta": "lerna publish --dist-tag beta --canary --preid beta -y",
+    "release:canary": "lerna publish --dist-tag canary --canary --preid canary -y",
+    "release:rc": "lerna publish --dist-tag rc --canary --preid rc -y",
+    "release:stable": "lerna publish --dist-tag stable minor -y"
```

Comment:
We should note this will continually update the second semver number so we will be going 5.14 -> 5.15 -> 5.16.
In the future if and when we do correct versioning we would need a way to do major versions when introducing breaking changes, just something to think about

Reply:
updated and added note